### DISCUSSION TITLE
Unable To Download ISO

### DISCUSSION THREAD

**Discussion by KKenui** [Security-Onion-Solutions/securityonion#3291] [Unsupported Versions]
I've been trying multiple ways to download https://download.securityonion.net/file/securityonion/securityonion-2.3.30.iso per the instructions but in never downloads more than 50MB then dies.

**Comment by KKenui** [comment]
Still unable to download the ISO and don't know how to get help. I just found out about this program and was excited to install and test for possible production but unable to even obtain the ISO for install.

**Reply by jertel** [reply]
The ISO is downloading for me without issues. Have you tried downloading the ISO using an another Internet provider, to see if the problem is with your ISP? Or tried using a different device?

**Reply by TOoSmOotH** [reply]
You can also just do a network based install as well. Just install Ubuntu 18.04 or CentOS 7 Minimal and clone the github.
